index.html

```diff
@@ -5,8 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    
+    <h1>About Erin Gurley</h1>
```

Comment:
It's good practice to wrap these top lines in a header tag for semantics/screenreaders